.github/workflows/ruby.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ["3.2", "3.3"]
+        ruby: ["3.3", "3.4"]
```

Comment:
Mylibrary is currently deployed on 3.2, so we should keep that until the deploy is on something newer.